### PR TITLE
feat: adding aks workshop

### PR DESCRIPTION
**Pull request by pauldotyu**
Adding AKS workshop which was originally delivered at MSBuild 2023 and redelivered at MS Reactor livestream.

### CODE REVIEW

File: package-lock.json
Comment (cmaneu) [inline package-lock.json:1]: @pauldotyu Not sure this file should be part of this PR

File: workshops/getting-started-with-aks/assets/banner.jpg
Comment (cmaneu) [inline workshops/getting-started-with-aks/assets/banner.jpg:1]: I don't see this image in the workshop. Is it useful?

**Review by cmaneu — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (pauldotyu) [reply package-lock.json:1]: Maybe not? I did a `npm install` to run the site locally and must have detected changes in packages. Will revert this change.

File: workshops/getting-started-with-aks/assets/banner.jpg
Comment (pauldotyu) [reply workshops/getting-started-with-aks/assets/banner.jpg:1]: Great point. I must have copied it over from the `/template/workshop` directory and forgot about it. I'll remove it.

**Review by cmaneu — APPROVED** [review]
LGTM 🚀

**Pull request merged**